common/components/filter/filter.stories.js

```diff
@@ -0,0 +1,86 @@
+import macroTemplate from './filter.stories.njk'
+
+export default {
+  title: 'Components / Filter',
+  component: macroTemplate,
+}
+
+const Template = data => `${macroTemplate(data)}`
+
+export const basic = Template.bind({})
+basic.args = {
```

Comment:
I wonder if we could make use of the existing [examples](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/098a1522281d1d26953114c2d534c5f80d1bb09e/common/components/filter/filter.yaml#L33) that we already define in the yaml config file for each component?

And have some kind of generic story creator for each component?